System.Runtime/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// ReSharper disable once CheckNamespace
+namespace System.Runtime.CompilerServices
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Why not adding the Intellisense comments?

Reply:
I was just copy/pasting from the reference code. I updated it to have the documentation from the web.